model/src/schema/schema.ts

```diff
@@ -235,6 +235,9 @@ const feeOptionSchema = joi
     payApiKey: [joi.string().allow("").optional(), multiApiKeySchema],
     paymentReferenceFormat: [joi.string().optional()],
     payReturnUrl: joi.string().optional(),
+    allowSubmissionWithoutPayment: joi.boolean().optional().default(true),
+    maxRetries: joi.number().optional().default(3),
+    customPayErrorMessage: joi.string().optional(),
```

Comment:
I Think customPayErrorMessage should have a default on it. Otherwise in the template, all that will be shown is the error and the retry button

Reply:
I've changed to `maxAttempts`. Retries implies one initial try + additional attempts. Attempts would just be "all the times the user has tried"